src/App.js

```diff
@@ -10,10 +12,35 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+export class App extends React.PureComponent {
+  state = {
+    selectedTabId: 'tab-2',
+  }
+
+  onTabSelected = (selectedTab) => {
+    this.setState({
+      selectedTabId: selectedTab,
+    });
```

Comment:
1. Пропиши умову, що якщо в тебе таба вже відкрита, то виконання ігнорується
2. Якщо ти вже називаєш event selectedTab, то чому б тоді  вже не назвати його selectedTabId, щоб скоротити запис?

Але, **ІМХО**, краще записати що це значення івенту, тоді читабельність збільшиться
```suggestion
  onTabSelected = (selectedTabId) => {
    this.setState({
      selectedTabId,
    });
```